engine/src/main/java/net/jqwik/engine/facades/ArbitrariesFacadeImpl.java

```diff
@@ -140,6 +140,59 @@ public StringArbitrary strings() {
 		return new DefaultStringArbitrary();
 	}
 
+	@Override
+	public Arbitrary<String> emails() {
```

Comment:
I know we are going full circle, but now is probably a good time to introduce class `DefaultEmailArbitrary`. An interface for it is not needed yet IMO.

Reply:
Is my basic implementation correct?